orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -28,6 +29,7 @@
 @Slf4j
 public class OrchestrationClient {
   private static final String DEFAULT_SCENARIO = "orchestration";
+  private static final String COMPLETION_ENDPOINT = "/v2/completion";
```

Comment:
(Question)

Can you imagine a use-case where someone would prefer the v1 version?